app/views/users/_follow.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= form_for (current_user.active_relationships.build, remote: true) do |f| %>
+    <div><%=hidden_field_tag :followed_id, @user.id %></div>
+    <% f.submit "Follow", class: "btn btn-large btn-primary" %>
```

Comment:
Your button will not show up because this the wrong way to call a BUTTON in Rails